Makefile

```diff
@@ -18,8 +18,23 @@
 TOP := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 SHELL := /bin/bash
 
+# Make sure GOPATH is set based on the executing Makefile and workspace. Will override
+# GOPATH from the env.
+export GOPATH= $(shell cd ../../..; pwd)
```

Comment:
I'm curious why we need to override the env variable

Reply:
Concerns about externally set wrong GOPATH (maybe including other things )